.github/workflows/tag-release.yml

```diff
@@ -54,10 +54,13 @@ jobs:
       - name: "Checkout ${{ github.ref }} ( ${{ github.sha }} )"
         uses: actions/checkout@v4
         with:
-          persist-credentials: false
-          tags: true
           fetch-depth: 0
 
+      - name: Use Node.js 20
```

Comment:
Oooh, fancy. Time to upgrade the repo to 18... maybe it'll all be on the same version one day :D

Reply:
yeah `supersetbot` requires 20 as I'm using some of the newer ecmascript features and didn't want to set up babel. If I remember well I think that was just the from `import ... from ...` synthax that force me up to 20. No way I was going to deal with `require()`